std/typecons.d

```diff
@@ -6582,17 +6582,17 @@ mixin template Proxy(alias a)
     static struct S
     {
         int foo() { return 12; }
-        mixin Proxy!foo;
+//        mixin Proxy!foo;
     }
     static class C
     {
         int foo() { return 12; }
         mixin Proxy!foo;
     }
-    S s;
-    assert(s + 1 == 13);
+//    S s;
+//    assert(s + 1 == 13);
```

Comment:
This removes the test for the normal code path too, so if we really decide we don't want to test this for DIP1000, I think it should be sth. like:

```
unittest
{
    static class C
...
}

version(DIP1000) {} else
unittest
{
   static struct S
}
```

Reply:
There isn't a standard DIP1000 version (perhaps there should be), but in any case the reasons these tests are commented out should be made clear in the code.